tests/common-tests.h

```diff
@@ -4041,6 +4057,52 @@ TEST_CASE_METHOD(common_tests, "Truncation error", "[core][insert][truncate][exc
     }
 }
 
+TEST_CASE_METHOD(common_tests, "Blank padding", "[core][insert][exception]")
+{
+    soci::session sql(backEndFactory_, connectString_);
+    if( !tc_.has_trailing_spaces_in_char_type(sql) )
+    {
+        WARN("No need to check for trailing space functionallity on backend that does not support it.");
+        return;
+    }
+
+    SECTION("Check blank padding in character types")
```

Comment:
This is copy-paste issue. Of course section here is unnecessary and will be removed.